packages/request-node/babel.config.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  presets: [['@babel/preset-env', { targets: { node: 'current' } }], '@babel/preset-typescript'],
+  plugins: ['@babel/plugin-transform-runtime', '@babel/plugin-transform-modules-commonjs'],
```

Comment:
`@babel/plugin-transform-runtime`
[Here](https://babeljs.io/docs/babel-plugin-transform-runtime) it says to also install `@babel/runtime` when using this plugin (for us, as a dev dependency as we only need it with Jest). It's already installed as an indirect dependency:
```
@babel/preset-env > @babel/plugin-transform-regenerator > regenerator-transform > @babel/runtime
```
Should we add it as a direct dependency?